helm/examples/metrics/elasticsearch_metrics.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Defines a elastic sink for reporting metrics to an ElasticSearch instance
+# Settings:
+#   type                                - sink type (must be elastic for ElasticSearch support)
+#   name                                - name for the sink instance
+#   settings.countMetricSuffix          - suffix for count metrics (default: count)
```

Comment:
This seems fine, but I'm not seeing the benefit of this. 
Will this setting apply a suffix to all count metrics across all components? What's the use case for using anything other than count? and is there a delimiter between the suffix and the rest of the metric name?

Reply:
Ok, so I think the answer is, the prefix should match the Elasticsearch dynamic mapping configuration in order to map the types/vales correctly. It still seems we could always prefix those types with the default value, but I don't see anything wrong w/ the extra flexibility